src/main/java/net/pms/dlna/LibMediaInfoParser.java

```diff
@@ -514,14 +534,30 @@ private static void getFormat(StreamType streamType, DLNAMediaInfo media, DLNAMe
 			format = FormatConfiguration.MP4;
 		} else if (value.contains("mjpg") || value.contains("m-jpeg")) {
 			format = FormatConfiguration.MJPEG;
-		} else if (value.contains("div") || value.contains("dx")) {
-			format = FormatConfiguration.DIVX;
+		} else if (
+				value.startsWith("div") ||
+				value.equals("dx50") ||
+				value.equals("dvx1")
+			) {
+				format = FormatConfiguration.DIVX;
+		} else if (value.startsWith("indeo")) { // Intel Indeo Video: IV31, IV32, IV41 and IV50
+			format = FormatConfiguration.INDEO;
+		} else if (value.equals("yuv")) {
+			format = FormatConfiguration.YUV;
+		} else if (value.equals("rgb") || value.equals("rgba")) {
```

Comment:
Ah, now I see the change. Yes, that helps. For the MJPEG case it would be perfect, here I think some further explanation is needed about these very unintuitive ```FormatConfiguration```s.